src/components/CatalogContext.js

```diff
@@ -15,9 +15,11 @@ class CatalogContext extends Component {
         theme,
         title,
         pages: pages.filter((p) => !p.hideFromMenu),
+        pagePaths: new Set(pages.map((p) => p.path)),
```

Comment:
I understand the reason behind this but I think it would be nice if we would warn the user that there are duplicate paths, if any. We ran into this problem with a lot of people editing the docs and would have appreciated a bit of help.